config/config.go

```diff
@@ -105,6 +105,8 @@ type Config struct {
 	// Number of seconds to keep retrying HTTP requests. Default is 300 (5 minutes)
 	RetryTimeoutSeconds int `name:"retry_timeout_seconds" auth:"-"`
 
+	HTTPTransport *http.Transport
```

Comment:
```suggestion
	HTTPTransport http.RoundTripper
```

`*http.Transport` is too concrete in this place. can't do that.

Reply:
@tcesnik-veza please also add a unit test for this change.